readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -37,6 +37,8 @@ class BaseSphinx(BaseBuilder):
 
     """The parent for most sphinx builders."""
 
+    sphinx_doctrees_dir = "_build/doctrees"
```

Comment:
This needs a comment I think explaining what it's for. Also seems like it should probably not be plural?

Reply:
I added a small comment based on the Sphinx's documentation. The plural form comes from the default value of this setting `.doctrees`. Note that I haven't changed the name of this directory in this PR, I just moved it to a class variable instead of hardcode it in multiple places.